deploy/operator.yaml

```diff
@@ -206,97 +206,106 @@ roleRef:
-                image
-              type: string
-            pathConfig:
-              description: PathConfig contains the information needed to build the
-                path where the PVs will be created.
-              properties:
-                path:
-                  type: string
-                useNamingPrefix:
-                  type: string
-              type: object
-          required:
-          - pathConfig
-          type: object
-        status:
-          description: HostPathProvisionerStatus defines the observed state of HostPathProvisioner
-          properties:
-            conditions:
-              items:
-                description: Condition represents the state of the operator's reconciliation
-                  functionality.
+  apiVersion: apiextensions.k8s.io/v1beta1
+  kind: CustomResourceDefinition
+  metadata:
+    name: hostpathprovisioners.hostpathprovisioner.kubevirt.io
+  spec:
+    group: hostpathprovisioner.kubevirt.io
+    names:
+      kind: HostPathProvisioner
+      listKind: HostPathProvisionerList
+      plural: hostpathprovisioners
+      singular: hostpathprovisioner
+    scope: Cluster
+    validation:
+      openAPIV3Schema:
+        description: HostPathProvisioner is the Schema for the hostpathprovisioners
+          API
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: HostPathProvisionerSpec defines the desired state of HostPathProvisioner
+            properties:
+              imagePullPolicy:
+                description: ImagePullPolicy is the container pull policy for the host
+                  path provisioner containers
+                type: string
+              pathConfig:
+                description: PathConfig is the path configuration of the path used to
+                  create PVs
                 properties:
-                  lastHeartbeatTime:
-                    format: date-time
-                    type: string
-                  lastTransitionTime:
-                    format: date-time
-                    type: string
-                  message:
+                  path:
+                    description: Path The path the directories for the PVs are created
```

Comment:
Maybe change to...
The provisioner will store PVs at this location on each node.

Reply:
Done